R/asymptoticTimings.R

```diff
@@ -32,15 +32,19 @@ asymptoticTimings <- function(e, data.sizes, max.seconds)
 
   timings.list <- list()
 
-  for(i in seq(along = data.sizes))
+  i <- 1
+
+  while(i <= length(data.sizes))
   {
     benchmarked.timings <- as.data.frame(microbenchmark(fun.obj(data.sizes[i])))
 
     benchmarked.timings$data.size <- data.sizes[i]
 
     timings.list[[i]] <- data.frame(benchmarked.timings$time, benchmarked.timings$data.size)
 
-    if(mean(benchmarked.timings$time) > time.limit) break else next
+    if(mean(benchmarked.timings$time) > time.limit) break
```

Comment:
again please move this condition into the while(), please avoid using break, it is confusing

Reply:
sure, done